model/model.proto

```diff
@@ -167,14 +175,9 @@ message ModelVersion {
 message ModelInfo {
   int32 id                       = 1;
   string name                    = 2;
-  bool optimized                 = 3;
-  string type                    = 4;
-  string framework               = 5;
-  string visibility              = 6;
-  repeated ModelVersion versions = 7;
-  string fullName                = 8 [(google.api.field_behavior) = OUTPUT_ONLY];
-  repeated string contents       = 9;
-  CVTask cv_task                 = 10;
+  string fullName                = 3 [(google.api.field_behavior) = OUTPUT_ONLY];
```

Comment:
let's change the name to `full_name` for conventional point of view, I'll also update pipeline as well